packages/pn-personafisica-webapp/src/components/Contacts/ContactsSummaryCards.tsx

```diff
@@ -55,13 +55,25 @@ const ContactsSummaryCard: React.FC<ContactsSummaryCardProps> = ({
       return t('summary-card.no-address');
     }
 
-    const contactsType = availableAddresses.reduce((acc, item) => {
-      // eslint-disable-next-line functional/immutable-data
-      acc[item.channelType] = t(`summary-card.${item.channelType}`);
-      return acc;
-    }, {} as { [key: string]: string });
+    const contactsType = availableAddresses.reduce(
+      (acc, item) => {
+        // eslint-disable-next-line functional/immutable-data
+        acc[item.channelType] = t(`summary-card.${item.channelType}`);
+        return acc;
+      },
+      {} as { [key: string]: string }
+    );
 
-    return Object.values(contactsType).join(', ');
+    if (
```

Comment:
This fix doesn't cover all cases.
Restore the old version of the file and define a sort function in the contacts.utility file. The sort order must be:

- before the addresses with senderId === 'default' and after the addresses with senderId !== 'default'
- otherwise before those addresses that have higher priority

The priority is (from highest to lowest): SERCQ, PEC, APPIO, EMAIL and SMS. After the sort function has been created, use it in the getDigitalAddresses.fulfilled and in updateAddressesList into the else block.

Remember to write test cases.

TIPS: for the priority's management you can use the addressesRelationships array. Here you can add the APPIO type and after define, for each element, a priority key of type number. The lowest priority will have value 0 and the highest will have 4. After that in your sort function you can use this array to check the priority of the single address